src/test/suite/fixtures/recorded/actions/drinkCap.yml

```diff
@@ -0,0 +1,27 @@
+spokenForm: drink cap
+languageId: typescript
+command:
+  actionName: editNewLineAbove
+  partialTargets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: c}
+  extraArgs: []
+marks:
+  default.c:
+    start: {line: 0, character: 0}
+    end: {line: 0, character: 5}
+initialState:
+  documentContents: "const myVariableValue = \"hello\";\r\n"
+  selections:
+    - anchor: {line: 1, character: 0}
+      active: {line: 1, character: 0}
+finalState:
+  documentContents: "\r\nconst myVariableValue = \"hello\";\r\n"
```

Comment:
Does VSCode auto-detect line ending type? If not then this will fail in some cases I believe